app/routes/session.py

```diff
@@ -132,7 +132,16 @@ def get_sign_out():
     """
     Signs the user out of eQ and redirects to the log out url.
     """
-    log_out_url = get_survey_config().account_service_log_out_url
+    todo = request.args.get("todo", "") == "True"
+    survey_config = get_survey_config()
+    if todo:
+        log_out_url = (
+            survey_config.account_service_todo_url
+            or survey_config.account_service_log_out_url
+        )
+    else:
+        # Future implementation when "todo" is not hardcoded on the template level
```

Comment:
I don't understand this.

Reply:
Ah I see what you mean. Ideally you would not always pass `true`, i.e for non-business, but I think for now, that is fine.

The logic just need's updating to something like:
```py
    log_out_url = survey_config.account_service_todo_url if todo else None

    if not log_out_url:
        log_out_url = survey_config.account_service_log_out_url
```

Reply:
Ok, added option 1 as I find it more readable, thanks.